design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -104,7 +99,7 @@ type DeleteNegotiableQuotesOutput {
     # Implementation Note: We don't make the deleted quotes accessible
     # because "deleted" means they're hidden from the storefront UI. They will
     # still be visible (and labeled as deleted) for a Seller in the admin
-    negotiable_quotes(
+    negotiableQuotes(
```

Comment:
```suggestion
    negotiable_quotes(
```

Should be a snake case per our conventions.

Reply:
@melnikovi Should we change to that snake case on line no. 8 also?

Reply:
On the first level it's camel case.